### DISCUSSION TITLE
Shipit.yml - where would it be found?

### DISCUSSION THREAD

**Discussion by cgmcnew** [Shopify/shipit-engine#1194] [Q&A] [answered]
I am looking for the shipit.yml file, but am not so sure where it would be found.
Can anyone point me in the direction of where it should be located?

Thank you

**Comment by benlangfeld** [comment, marked as answer]
You would write it...

**Reply by cgmcnew** [reply]
Thank you that is what I was thinking was the case here...
Does it matter where I place the shipit.yml file? Where is the reference for where it is expected (incase I might have missed it as I browsed through)

Thank you

**Reply by casperisfine** [reply]
Shipit will look for a `shipit.yml` at the root of the deployed repository.
